code/tools/make_munki_mpkg.sh

```diff
@@ -32,18 +32,19 @@ fi
 
 usage() {
     cat <<EOF
-Usage: `basename $0` [-i id] [-r root] [-o dir] [-c package]"
+Usage: `basename $0` [-i id] [-r root] [-o dir] [-c package] [-s cert]"
 
     -i id       Set the base package bundle ID
     -r root     Set the munki source root
     -o dir      Set the output directory
     -c package  Include a configuration package (NOT CURRENTLY IMPLEMENTED)
+    -s cert     Sign distribution package with signing certificate from keychain
```

Comment:
Might want to better describe the needed input string; this is going to confuse people